### PR TITLE
Add parse_yaml function

### PR DESCRIPTION
**Pull request by clayscode**

### CODE REVIEW

**Comment by clayscode** [comment]
Hmm, `SELECT parse_yaml(...) FROM scope()` seems to work correctly, but selecting subelements returns null for everything.

**Comment by clayscode** [comment]
Alright, I believe I've fixed the issue. My PR, however, adds an additional dependency to https://github.com/kubernetes-sigs/yaml which is an MIT licensed repository with contributors.

File: go.sum [hunk 1]
```diff
@@ -241,8 +244,11 @@ github.com/golang/protobuf v1.4.0-rc.4.0.20200313231945-b860323f09d0/go.mod h1:W
 github.com/golang/protobuf v1.4.0/go.mod h1:jodUvKwWbYaEsadDk5Fwe5c77LiNKVO9IDvqG2KuDX0=
 github.com/golang/protobuf v1.4.1/go.mod h1:U8fpvMrcmy5pZrNK1lt4xCsGvpyWQ/VVv6QDs8UjoX8=
 github.com/golang/protobuf v1.4.2/go.mod h1:oDoupMAO8OvCJWAcko0GGGIgR6R6ocIYbsSw735rRwI=
-github.com/golang/protobuf v1.4.3 h1:JjCZWpVbqXDqFVmTfYWEVTMIYrL/NPdPSCHPJ0T/raM=
 github.com/golang/protobuf v1.4.3/go.mod h1:oDoupMAO8OvCJWAcko0GGGIgR6R6ocIYbsSw735rRwI=
+github.com/golang/protobuf v1.5.0 h1:LUVKkCeviFUMKqHa4tXIIij/lbhnMbP7Fn5wKdKkRh4=
```
Comment (scudette) [inline go.sum:248]: maybe run `go mod tidy`

**Review by scudette — COMMENTED** [review]
We do need some tests here but I can add them later if you like.

**Pull request merged**